app/models/list.rb

```diff
@@ -29,4 +32,14 @@ def add_item params
 
     item
   end
+
+  def as_json(*)
+    super({
+      only: [:id, :name],
+      include: :items
```

Comment:
This is going to pass all of the attributes in `Item` record.  If you don't want this to happen, you either need to switch to hash-syntax and pass an `only:` key/value pair to limit the columns, or implement an `Item#as_json` method.

Reply:
Completed with [this commit](https://github.com/Brian-Demon/shopping-list/pull/61/commits/00ba5a73d2fcea5427c2ab2640ef8bf3589ec04e).